melodic/distribution.yaml

```diff
@@ -4356,6 +4356,17 @@ repositories:
       url: https://github.com/ros-drivers/omron.git
       version: kinetic-devel
     status: maintained
+  omronsentech_camera:
+    doc:
+      type: git
+      url: https://github.com/ose-support-ros/omronsentech_camera.git
+      version: master
+    source:
+      test_commits: false
```

Comment:
Please remove the `test_commits` line here; false is the default, and we only need to add the line if we want to enable it.

Reply:
Hi,

The test_commits has been removed.

Thanks,

OSE ROS Support.

Reply:
Thanks!